Makefile

```diff
@@ -83,12 +82,14 @@ deb-v2-x86:
 	$(MAKE) build/mackerel-plugin GOOS=linux GOARCH=amd64
 	cp build/mackerel-plugin packaging/deb-v2/debian/
 	cd packaging/deb-v2 && debuild --no-tgz-check -rfakeroot -uc -us
+	git clean -f -d ./packaging
```

Comment:
I understand this *git clean* is there to keep **debug.BuildInfo** as clean.

If so, I prefer that to put the line "before" build because the build process will be able to exit unexpectedly.

Reply:
I did. 
https://github.com/mackerelio/mackerel-agent-plugins/pull/1278/commits/02bc4acf36f7d5df72de50a8b1b15b7ad8808854

also added same clean process on before tar builds
https://github.com/mackerelio/mackerel-agent-plugins/pull/1278/commits/3cd171a91f1fbabc03742788a8b5513d8a7233bf